shared/wallets/wallet/settings/container.tsx

```diff
@@ -29,7 +52,11 @@ const mapStateToProps = state => {
   const canSubmitTx = account.canSubmitTx
   const thisDeviceIsLockedOut = account.deviceReadOnly
   const inflationDest = Constants.getInflationDestination(state, accountID)
-  const externalPartners = Constants.getExternalPartners(state, accountID).toArray()
+  const externalPartners = prepareExternalPartners(
```

Comment:
do not make a call like this in mapstate  you'll thrash every time, belongs in merge or needs to be memoized